src/elements/content-sidebar/activity-feed/comment/components/BaseCommentMetadata.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { ACTIVITY_TARGETS } from '../../../../common/interactionTargets';
+import { PLACEHOLDER_USER } from '../../../../../constants';
+import ActivityStatus from '../../common/activity-status';
+import ActivityTimestamp from '../../common/activity-timestamp';
+import Avatar from '../../Avatar';
+import IconAnnotation from '../../../../../icons/two-toned/IconAnnotation';
+import UserLink from '../../common/user-link';
+
+import type { FeedItemStatus } from '../../../../../common/types/feed';
+import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../../common/flowTypes';
+import type { User } from '../../../../../common/types/core';
+
+import messages from '../messages';
+
+import './BaseCommentMetadata.scss';
+
+export interface BaseCommentMetadataProps {
+    annotationActivityLink?: React.Element<any> | typeof undefined;
+    created_at: string | number;
+    created_by: User;
+    getAvatarUrl: GetAvatarUrlCallback;
+    getUserProfileUrl?: GetProfileUrlCallback | typeof undefined;
+    status?: FeedItemStatus | typeof undefined;
+}
+
+export const BaseCommentMetadata = ({
+    annotationActivityLink,
+    created_at,
+    created_by,
+    getAvatarUrl,
+    getUserProfileUrl,
+    status,
+}: BaseCommentMetadataProps) => {
+    const createdByUser = created_by || PLACEHOLDER_USER;
+    const createdAtTimestamp = new Date(created_at).getTime();
+
+    return (
+        <div className="bcs-BaseComment-Metadata">
+            <div className="bcs-BaseComment-Metadata-Avatar">
+                <Avatar
+                    getAvatarUrl={getAvatarUrl}
+                    user={createdByUser}
+                    badgeIcon={
+                        annotationActivityLink && (
+                            <IconAnnotation title={<FormattedMessage {...messages.annotationBadge} />} />
+                        )
+                    }
+                />
+            </div>
+            <div className="bcs-BaseComment-Metadata-Data">
+                <div className="bcs-Comment-headline">
```

Comment:
SUIT conventions call for every class name to be scoped to the root name of the component it's in. In this case, it would be `bcs-BaseCommentMetadata-headline`.

Reply:
Hey, I see you're looking at this. FYI, still a work-in-progess, but I do appreciate it. Re: this class, it's not a new class that I'm creating. It's a holdover from Comment.